packages/react-checkbox/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -1,143 +1,139 @@
 import * as React from 'react';
-import { render, RenderResult, fireEvent, screen } from '@testing-library/react';
+import { render, fireEvent } from '@testing-library/react';
 import { Checkbox } from './Checkbox';
 import { isConformant } from '../../common/isConformant';
 import { resetIdsForTests } from '@fluentui/react-utilities';
 
 // TODO: add more tests here, and create visual regression tests in /apps/vr-tests
 
 describe('Checkbox', () => {
-  let renderedComponent: RenderResult | undefined;
```

Comment:
What's the reasoning for removing the cleanup code?

Reply:
It's done automatically by testing-library/react: https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
yeah, it was manual up to some point, but not testing-library does this automagically